docs/partial_source/deep_dive/17_ivy_frontends_tests.rst

```diff
@@ -35,6 +35,12 @@ We assume knowledge of hypothesis data generation strategies and how to implemen
 
 * :code:`np_frontend_helpers.test_frontend_function()` behaves identical to :code:`helpers.test_frontend_function()` but handles NumPy's optional :code:`where` argument
 
+**Useful Notes**
+
+* There are more :code:`dtype` sets than :code:`valid_float_dtypes`, there is also :code:`valid_int_dtypes`, :code:`valid_numeric_dtypes` and many more! It's important when writing Ivy Frontend tests to pick the most appropriate :code:`dtype` set to ensure thorough testing!
+
+* The :code:`test_frontend_function` argument :code:`fn_tree` refers to the frontend function's reference in its native namespace not just the function name. For example :code:`lax.tan` for Jax or :code:`nn.functional.relu` for PyTorch.
```

Comment:
For example :code:`lax.tan` is needed for some functions in Jax, :code:`nn.functional.relu` is needed for some functions in PyTorch etc.